src/app/pages/forms-and-validation/fix-length-passcode/fix-length-passcode.component.ts

```diff
@@ -91,6 +91,7 @@ export class FixLengthPasscodeComponent implements OnInit, AfterViewInit {
                     this.showLoading = false;
                     this.showDoneIcon = false;
                     this.passcodeInput.nativeElement.focus();
+                    this.passcodeInput.nativeElement.select();
```

Comment:
Could you please add this feature in verify on submit? https://github.com/pxblue/angular-design-patterns/blob/dev/src/app/pages/forms-and-validation/verify-on-submit/verify-on-submit.component.ts#L73.
As per design need to highlight invalid input in `verify on submit` form.